packages/ui/src/Tabs/index.stories.tsx

```diff
@@ -34,6 +33,6 @@ export const Basic: Story = {
 
     const onChange = (value: { toString: () => string }) => updateArgs({ value });
 
-    return <SegmentedPicker {...props} onChange={onChange} />;
+    return <Tabs {...props} onChange={onChange} />;
```

Comment:
suggestion (non-blocking, worth implementing): expose `TabsContent` component from Radix and somehow allow the `children` for `Tabs`.

If you inspect the markup of the [Radix Tabs example](https://www.radix-ui.com/primitives/docs/components/tabs), you can see that `Tabs.Content` components are nested inside the `Tabs.Root`. It allows Radix to set useful attributes like `data-state="active"` and `role="tabpanel"`. Plus, there is a rendering logic involved that all `[role="tabpanel"]` divs are rendered but inactive divs have empty contents. All of this corresponds to W3C standards.

Developers might and probably will forget to implement this, so it would be useful to help them (and ourselves)

Reply:
Good call — will look into it. I'm not sure that `<TabsContent />` will be appropriate in our case, since we won't have multiple tab contents rendered on the page at the same time — `<Tabs />` will usually be used for routing, I believe. But I created [a ticket to look into it](https://github.com/prax-wallet/web/issues/131) nonetheless.